packages/bot-skeleton/src/scratch/blocks/Advanced/List/lists_split.js

```diff
@@ -71,13 +68,23 @@ Blockly.Blocks.lists_split = {
         }
 
         this.initSvg();
-        this.render(false);
+        // kept this commented to fix backward compatibility issue
+        //this.render(false);
```

Comment:
can u check if there is a clear method that replaces render false?

Reply:
checked this and we don't require this anymore